README.md

```diff
@@ -6,10 +6,26 @@ This gem provides a Ruby API for the [data services API](https://github.com/epim
 
 Add this line to your application's Gemfile:
 
-    gem 'data-api'
+```ruby
+gem 'data-api'
+```
 
 And then execute:
 
-    $ bundle
+```sh
+bundle
+```
 
-Also, an API URL needs to be provided for the Service class in order for the gem to work
\ No newline at end of file
+Also, an API URL needs to be provided for the Service class in order for the gem
```

Comment:
An explanation of why here might be useful?

Reply:
There's no fallback default value in the gem for the API URL, so if not specified the gem won't work. I'm gonna rephrase this to something like "you need to set <this> as an environment variable" so it's more clear to the gem user